### ISSUE TITLE
Manual jurisdiction implementations to test.

### ISSUE THREAD

**Issue by santisiri** [DemocracyEarth/sovereign#305]
Create initial digital jurisdictions on public instance for:

- [democracymars.org](democracymars.org)
- [partido.red](partido.red)
- [good.country](good.country)
- [womensmarch.org](womensmarch.org)

ccing @paulamlb

**Comment by santisiri** [comment]
> Should I include tentative deadlines?

yes, ideally a dashboard that serves as a pipeline to follow all ongoing projects would be great.

**Comment by paulamlb** [comment]
> yes, ideally a dashboard that serves as a pipeline to follow all ongoing projects would be great.

working on it, will invite you asap.

**Comment by paulamlb** [comment]
I have a question: can a url link to more then one hashtag? Ex: mexico.democracy.earth links to all poosts with #mexico and #presentesmx (event's hashtag).

**Comment by santisiri** [comment]
Yes, but right now that won't work very well right now. There's a minor bug when generating the link for the tag. I'll try to review and fix for this milestone.

**Comment by paulamlb** [comment]
Can we start creating manual jurisdictions to leverage every single event we are speaking at? 

For example, I'm speaking in Sao Paulo and Bahia this month, Herb is speaking at Northwestern University and Chicago this week. If it's an easy thing to set-up we set-up questions to show there in advance and then in each of the talks point people to the URLs (saopaulo.democracy.earth , bahia.democracy.earth , northwestern.democracy.earth , chicago.democracy.earth ) .

Alternatively - if it's too much work - we can set up the questions and point people to vote.democracy.earth/tag/saopaulo .... Let me know, and also let me know how you would like to be notified (to-do on basecamp can be a way).

**Comment by santisiri** [comment]
it's not much work, but it's still a manual thing where I configure the subdomain DNS. we should work on a feature on eventually automating this at the product level.

**Comment by santisiri** [comment]
implemented:
- [mars.democracy.earth](mars.democracy.earth)
- [argentina.democracy.earth](argentina.democracy.earth)
- [brazil.democracy.earth](brazil.democracy.earth)
- [venezuela.democracy.earth](venezuela.democracy.earth)
- [mexico.democracy.earth](mexico.democracy.earth)
- [women.democracy.earth](women.democracy.earth)

the ones connected to geographical region will show proposals from all of those who have in their profile that jurisdiction set. `women` i kept it in a tag, would be interesting to eventually just have a `/gender` in place. 

this all follows what discussed on #319.

**Comment by santisiri** [comment]
[mars included among jurisdictions](https://github.com/DemocracyEarth/sovereign/commit/e7e26c1972d08df569b59d56cc00f36023eb9cd3)

**Comment by paulamlb** [comment]
unitedcitizens.democracy.earth for the http://thefourthgroup.org/

We will be starting debates in this URL mid-June.

**Comment by paulamlb** [comment]
unitedcitizens.democracy.earth for the http://thefourthgroup.org/

It should point to: https://vote.democracy.earth/peer/unitedcitizens

We will be starting debates in this URL mid-June.

**Comment by paulamlb** [comment]
create goodcountry.democracy.earth pointing to https://vote.democracy.earth/peer/thegoodcountry

by tomorrow (Sat. 19/05)

**Comment by santisiri** [comment]
done.

**Comment by paulamlb** [comment]
if possible, change brazil.democracy.earth to brasil.democracy.earth (with an "s" instead of "z") 

by Wednesday 05/23

**Comment by paulamlb** [comment]
create saopaulo.democracy.earth

**Comment by paulamlb** [comment]
yale.democracy.earth - #yale 
ucberkeley.democracy.earth - #ucberkeley
uva.democracy.earth - #uva 
cornell.democracy.earth - #cornell 
northwestern.democracy.earth - #northwestern

**Comment by paulamlb** [comment]
@santisiri can you get northwestern.democracy.earth done by tomorrow? 
@SFSandra it would be great to announce the url with Robbie's interview once it's done, Robbie has his questions lined up to be posted as soon as it's ready.

**Comment by SFSandra** [comment]
@paulamlb is there an ETA on these?

**Comment by paulamlb** [comment]
Removed northwestern.democracy.earth from the list.

**Comment by santisiri** [comment]
Next time request to reopen the issue @paulamlb, it was closed.

**Comment by paulamlb** [comment]
create ciudadanosunidos.democracy.earth for https://vote.democracy.earth/peer/ciudadanosunidos

**Comment by paulamlb** [comment]
create vob.democracy.earth for https://vote.democracy.earth/peer/voiceofblockchain @santisiri

**Comment by paulamlb** [comment]
create tejidosocial.democracy.earth for https://vote.democracy.earth/peer/tejidosocial

**Comment by santisiri** [comment]
done.

**Comment by paulamlb** [comment]
academy.democracy.earth for https://vote.democracy.earth/tag/democracyacademy

**Comment by SFSandra** [comment]
@paulamlb I was thinking it would be good to post the Democracy Academy program at  https://vote.democracy.earth/tag/democracyacademy, i.e. an introductory message with a post/link for each unit, serves not only as an added channel for people to go through the academy curriculum, but also a place for feedback, discussion, debate.  You could 'close the loop' by adding a question at the end of each unit, i.e.  "Was this unit effective?  Rate your experience or leave a comment." that links to an associated poll on https://vote.democracy.earth/tag/democracyacademy. What do you think?

**Comment by paulamlb** [comment]
Yes @SFSandra I'm thinking on these lines. For now I think our platform makes sense as a forum. I want people to have to sign-up to get the course, as I described to you some days ago. But if/when we have markup and no character limit it will be wonderful to make it all available there!!

**Comment by leaogama** [comment]
Alguem entende portugues?
